docs/tutorials/connecting-to-the-internet-via-usb.md

```diff
@@ -135,6 +135,75 @@ If you are using an older version, please upgrade.
 
         {% include /util/screenshot.html source="/images/brickman/wired-status-online-connect-automatically-selected.png" %}
 
+
+*   {: tab="Debian_commandline"}{% include /style/icon.html type="info" %}
+    These instructions were written using Debian look-alikes for those wanting to know what happens under the hood.
+    The commands need to be executed by or as root.
+    {:class="alert alert-info"}
+
+    1.  Connect the USB cable to your computer and its mini-B plug to the EV3 (slot marked "PC").
+
+    2.  Verify that USB tethering is disabled: "Wireless and Networks" > "Tethering.
+
+    3.  Set default Linux network settings: "Wireless and Networks" > "All Network Connections" > "Wired" >
+        "IPv4" > "Change" > "Load Linux Defaults".  The EV3 should get address 10.42.0.3, gateway 10.42.0.1.
+
+    4.  Verify if the EV3 is recognized by your Linux PC:  "lsusb" should show
+        
+        `Bus 001 Device 002: ID 0525:a4aa Netchip Technology, Inc. Linux-USB CDC Composite Gadge (Ethernet and ACM)`
```

Comment:
OK, I've changed my mind (c.f. #308).

In this pull request though, I would like to have a blank line before the start of each code fence.